playwright.config.js

```diff
@@ -0,0 +1,15 @@
+const { defineConfig } = require('@playwright/test');
+
+module.exports = defineConfig({
+  testDir: './src/__tests__/e2e',
+  webServer: {
+    command: 'npm run dev',
+    port: 3000,
+    timeout: 120 * 1000,
+    reuseExistingServer: !process.env.CI,
+  },
+  use: {
+    baseURL: 'http://localhost:3000',
+    headless: true,
+  },
```

Comment:
Пропоную додати секцію projects для явного визначення браузерів (Chromium, Firefox, WebKit) для кросбраузерного тестування. І ще можна додати репортера для генерації детальних звітів про результати тестів.
```suggestion
  },
  reporter: [
  ['list'],
  ['html', { outputFolder: 'playwright-report', open: process.env.CI ? 'never' : 'on-failure' }],
],
```